### DISCUSSION TITLE
openldap upgrade version number issue

### DISCUSSION THREAD

**Discussion by Tpojka** [Homebrew/discussions#61] [Everyday usage]
I wasn't quite sure should I report this as issue itself so better to ask here. 
Although this seems like version number issue to me [terminal echo only, though I assume]:

![Image of Yaktocat](https://i.imgur.com/LFbfc6G.png)

(after upgrading) through `brew update && brew upgrade && brew cleanup`

It just looks weird `Upgrading openldap 2.4.54 -> 2.4.54`

**Comment by Rylan12** [comment]
Is this an issue that happens regularly? I agree it's strange but it does look like the correct thing happened (as the directory it removed was 2.4.53). I can't reproduce locally.

I'd say if it happens often or seems like a recurring bug feel free to open an issue (as long as you fill out the template the worst that can happen is the issue is closed).

**Reply by Tpojka** [reply]
@Rylan12 

I am regularly updating/upgrading brew packages. This is first time my eye caught something similar. 
Maybe if there is relation with patch number instead of minor semver number (i.e dir path is related to patch) it should be considered that way on output. 
Not sure though.
